sdk/nodejs/tests/runtime/registrations.spec.ts

```diff
@@ -28,15 +28,14 @@ describe("runtime", () => {
         describe("register", () => {
             const tests = [
                 { name: "wildcard version", version: undefined },
-                { name: "blank version", version: "" },
```

Comment:
This isn't a legal value at all - it happened to throw for an unrelated reason inside `sameVersion` - but that wasn't really what this test was trying to test.